openshift/templates/agent/agent-deploy.bc.test.param

```diff
@@ -12,14 +12,14 @@
 # SUFFIX=-bc
 # APP_NAME=orgbook
 # APP_GROUP=aries-vcr
-APPLICATION_DOMAIN=orgbook-agent-test.apps.silver.devops.gov.bc.ca
-APPLICATION_ADMIN_DOMAIN=orgbook-agent-admin-test.apps.silver.devops.gov.bc.ca
+APPLICATION_DOMAIN=agent-test.orgbook.gov.bc.ca
+APPLICATION_ADMIN_DOMAIN=agent-admin-test.orgbook.gov.bc.ca
 # ROUTE_TIMEOUT=120s
 # IMAGE_NAMESPACE=8ad0ea-tools
 # NAMESPACE_NAME=8ad0ea
 TAG_NAME=bc-test
 # AGENT_LABEL_NAME=OrgBook BC Agent
-AGENT_BASE_URL=https://orgbook-agent-test.apps.silver.devops.gov.bc.ca
+AGENT_BASE_URL=https://agent-test.orgbook.gov.bc.ca
```

Comment:
This should be `https://bc-reg-agent-test.orgbook.gov.bc.ca`

Reply:
updated